### DISCUSSION TITLE
Предложения по рейтингу смартапов

### DISCUSSION THREAD

**Discussion by Genenko-Svetlana** [sberdevices/salute-issues#264] [Catalog]
Приглашаем к обсуждению!
Недавно у нас было важное обновление - возможность оценки смартапов и формирование рейтинга.

Поделитесь, пожалуйста, опытом использования данного функционала, а также мнениями и идеями!

**Comment by subnetsRU** [comment]
Опыт использования: Сначала вообще не понятно почему поле с рейтингом отсутствует. Потом становится понятно, что надо запустить апп и только потом появляется возможность оценки.

В итоге две проблемы:
* не понятно что там вообще есть какой-то рейтинг. Ничего не указывает на это, если апп не был запущен, что после там появится некий рейтинг.
* можно просто запустить и закрыть апп и этого уже достаточно, чтобы начать оценивать.

Далее, разработчик не видит ни кол-во оценок, ни когда они были поставлены, ни какие они. Он видит не больше чем пользователь.

Вообщем моё ИМХО, текущая система никуда не годится и её появление в таком виде меня расстроила/опечалила. Уж простите, но это моё мнение и оно может отличаться от мнения других.

Когда-то со Всеволодом мы обсуждали этот вопрос. Оценивать апп без оставления внятного комментария И возможности разработчику ответить на этот комментарий пользователя это совсем не хорошо. В итоге, как пример я приводил систему оценки у "Я". Она никакая именно по этим причинам.
Можно оставить оценку без комментария, невозможно ответить на комментарий. Итогам невозможно оспорить/уточнить у пользователя причин:
* оценка 1, комментарий "Ха, я тупой!"
* оценка 1, комментарий "Ыц2"
* оценка 1, комментарий "Ничего не знаем"
* не у так далее и в таком же духе

Оценки без обратной связи с пользователем - ужасно.

Посему идеи:
* показывать "плашку"/заглушку на том месте где будет отображаться рейтинг
* пользователь должен взаимодействовать с апом более длительное время перед тем как сможет поставить оценку
* разработчику нужно видеть список оценок, даты, комментарии
* разработчику нужна обратная связь с пользователем
* разработчику нужна некая привязка отзыва хотя бы к последней сессии с пользователем

**Comment by sergeyzhirkov** [comment]
Самый большой недостаток, как мне кажется, что пользователю приходится возвращаться на страницу смартапа, чтобы оценить его.

То есть нельзя сразу после использования  сказать "поставь 5 звёзд". В мобильных приложениях (сбол, салют) при выходе из смартапа выбрасывает и из страницы со смартапом (где команда Запусти и оценки). Скорее всего, если смартап понравился и запомнился, в следующий раз пользователь будет запускать его фразой "запусти + название", соответственно на страницу смартапа не попадёт и не проголосует за него.

У меня была идея - при выходе из смартапа сообщать пользователю, что он может за него проголосовать и просить поставить 5 звёзд, но решил что это уж как-то совсем не красиво будет :)

**Comment by Genenko-Svetlana** [comment]
@Avalpetrova

**Reply by chernov2021** [reply]
Хочется иметь возможность в смартапе попросить пользователя поставить оценку. Сможете реализовать такое или подсказать как сделать самим?

**Comment by Genenko-Svetlana** [comment]
@Avalpetrova

**Comment by Genenko-Svetlana** [comment]
@chernov2021 В ближайшее время такая возможность будет реализована, уже работаем )))

**Reply by chernov2021** [reply]
1. А сроки есть уже, когда будет готово? 2. Как работают thumb up|down в саджестах внутри смартапов? Они как-то влияют на оценку? Есть ли по ним доки?
